Source/Scene/Cesium3DTileset.js

```diff
@@ -264,6 +264,7 @@ define([
 
         this._maximumScreenSpaceError = defaultValue(options.maximumScreenSpaceError, 16);
         this._maximumNumberOfLoadedTiles = defaultValue(options.maximumNumberOfLoadedTiles, 256);
+        this._maximumMemoryUsage = defaultValue(options.maximumMemoryUsage, 256);
```

Comment:
Both of these are private right. None of the privates have documentation.

Will make the change to 512.